src/components/modals/PlanMergeRequestsModal.svelte

```diff
@@ -223,9 +230,9 @@
                       {planMergeRequest.type === 'outgoing' ? 'Withdraw' : 'Begin Review'}
                     {/if}
                   </button>
-                {:else if planMergeRequest.status === 'in-progress'}
+                {:else if planMergeRequest.status === 'in-progress' && planMergeRequest.plan_snapshot_supplying_changes.plan}
                   <button
-                    on:click={() => goto(`${base}/plans/${planMergeRequest.plan_receiving_changes.id}/merge`)}
+                    on:click={() => goto(`${base}/plans/${planMergeRequest.plan_receiving_changes?.id}/merge`)}
```

Comment:
Would it be more clear to disable/hide the button if `plan_receiving_changes` is undefined? If it is somehow able to get to this point, the `goto()` will potentially go to a `/plans/undefined/merge` route.

Reply:
Possibly though if a merge request is in progress then the plan receiving changes has to exist and can't even be deleted since the plan is locked?

Reply:
I think I might prefer to go off of what the type says than what pragmatically happens in this case. It would be one less thing to have to remember to update if this ever does change.